src/obb/server/routes.clj

```diff
@@ -0,0 +1,13 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.routes
+  "Specifices the routes and associates handlers"
+  (:require
+    [compojure.core :as compojure :refer [GET]]
+    [compojure.route :as route]
+    [obb.server.handlers.index :as index]))
+
+(compojure/defroutes public-routes
+  "The routes available to be served, that don't need auth"
+  (GET "/" request (index/handler request))
+  (route/not-found "No such page."))
```

Comment:
That's standard template stuff. I changed it to return a result map, just like the other handlers.

I know that you feel that the status is enough, but for me status + result body makes sense.